src/cart/src/cart/crt_group.c

```diff
@@ -189,23 +181,16 @@ rm_op_rec_addref(struct d_hash_table *hhtab, d_list_t *rlink)
 	struct crt_rank_mapping *rm = crt_rm_link2ptr(rlink);
 
 	D_ASSERT(rm->rm_initialized);
-	D_MUTEX_LOCK(&rm->rm_mutex);
-	rm->rm_ref++;
-	D_MUTEX_UNLOCK(&rm->rm_mutex);
+	atomic_fetch_add(&rm->rm_ref, 1);
```

Comment:
It's the same under the covers I guess but why not atomic_inc ?

Reply:
Hmm. Just to use small subset of all available operations. If we will need to increase to more than 1 we will change only number instead of operation.